src/components/molecules/ScheduleEventListNG/ScheduleEventListNG.tsx

```diff
@@ -32,7 +36,31 @@ export const ScheduleEventListNG: React.FC<ScheduleEventListNGProps> = ({
     setAllEvents([...daysEvents]);
   }, [daysEvents, setAllEvents]);
 
-  const liveEvents = useMemo(() => allEvents.filter(isEventLive), [allEvents]);
+  const { currentVenue } = useRelatedVenues({
+    currentVenueId: venueId,
+  });
+  const roomList = allEvents.map((el) => {
+    const [roomData] =
+      currentVenue?.rooms?.filter((room) => room.title === el?.room) || [];
+    return roomData;
+  });
```

Comment:
not sure we need to tho, we'll have online attendees prop only for live events, so modifying all events via `prepareForSchedule` seems unnecessary to me

Reply:
you can leave not live events without changes, and add attendees prop only to live ones.
By doing that you won't need to do props drilling (venueId, isLive) because you'll be able to use the information in the event object.
what would you say?

Reply:
fixed 6026561